.github/workflows/basics.yml

```diff
@@ -4,7 +4,7 @@ on:
       - main
   pull_request:
 
-name: Continuous integration - Serial Keel
+name: Serial Keel - Build
```

Comment:
Is this the name for all the jobs below?
Then "build" is too specific since we do build, test, lint, etc

Reply:
Ah ok! Cool :slightly_smiling_face: 
I think `Serial Keel - CI` would be ok.